priv/repo/migrations/20231113160043_add_charlie_cards_table.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Phil.Repo.Migrations.AddCharlieCardsTable do
+  use Ecto.Migration
+
+  def change do
+    create table("charlie_cards") do
+      add :batch_number, :integer, null: false
+      add :batch_sequence_number, :integer, null: false
+      add :card_valid_from, :utc_datetime, null: false
+      add :card_valid_until, :utc_datetime, null: false
```

Comment:
I wonder if we will know this always? In my charlie the valid until is set only once we cancel, is that the same here?

Reply:
We get this from the manufacturer. This is the date when the card itself will stop working...like, won't scan anymore. There is also the product valid until which is when the product expires. Youth Pass is only good for a calendar year. If we run into situations where we don't know this, we can remove it.

Reply:
Ahh gotcha